witchcraft/internal/middleware/request_test.go

```diff
@@ -43,7 +43,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// TestCombinedMiddleware tests the combined behavior of the NewRequestContextLoggers and NewRequestExtractIDs request
+// TestCombinedMiddleware tests the combined behavior of the NewRequestTelemetry and NewRequestExtractIDs request
```

Comment:
nit: This test can just be renamed to `TestRequestTelemetryMiddleware`

Reply:
thanks! updated